sql/core/src/main/scala/org/apache/spark/sql/execution/ui/SparkPlanGraph.scala

```diff
@@ -202,8 +197,8 @@ private[ui] class SparkPlanGraphCluster(
     val labelStr = if (duration.nonEmpty) {
       require(duration.length == 1)
       val id = duration(0).accumulatorId
-      if (metricsValue.contains(id)) {
-        name + "\n \n" + duration(0).name + ": " + metricsValue(id)
+      if (metricsValue.contains(duration(0).accumulatorId)) {
```

Comment:
Please keep `metricsBalue.contains(id)` as is. Let's avoid calling `duration(0).accumulatorId`  twice.